Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -56,9 +56,14 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
             var content = new Y.eZ.Content(),
                 version = new Y.eZ.Version(),
                 type = this.get('contentType'),
-                defaultFields = {};
+                defaultFields = {},
+                contentTypeNames,
+                contentTypeName;
 
-            content.set('name', 'New "' + this.get('contentType').get('names')['eng-GB'] + '"');
+            contentTypeNames = type.get('names');
+            contentTypeName = contentTypeNames[this.get('app').get('defaultLanguageCode')] || contentTypeNames[0];
```

Comment:
This seems incomplete, we need some js service that behaves like we do on PHP side:
Loop over prioritized languages and pick first match, and if none fallback to main language of the given object.